### ISSUE TITLE
[$250] Screen Reader: Global: Focus is lost when returning to the previous screen

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77411]
## Action Performed:
Precondition: User is logged in

1. Using TalkBack on Android, open the app
2. Swipe right to focus the Account tab and double-tap to activate it
3. Swipe right to focus the Profile button and double-tap to activate it
4. Swipe right to focus the Display name menu item and double-tap to activate it
5. Swipe to focus the Back button and double-tap to activate it
6. Note the focus behavior when the Profile screen loads

This is a global issue when navigating to previous screens

## Expected Result:
After navigating back to the previous screen, TalkBack focus should return to the element that triggered the navigation (e.g., the "Display name" menu item) and it should be announced.

Users should be able to continue interacting without losing context.

## Actual Result:
After navigating back to the previous screen, focus is completely lost and TalkBack announces nothing.

Users may be unaware that a screen-change occurred.

## Area issue was found in:
Profile

## Failed WCAG checkpoints
WCAG 2.4.3 Focus Order
https://www.w3.org/WAI/WCAG22/Understanding/focus-order

## User impact:
People with mobility impairments who must rely on keyboard access for operating a page benefit from a logical, usable focus order. People with vision impairments or disabilities that make reading difficult can become disoriented when tabbing takes focus someplace unexpected or when they cannot easily find the content surrounding an interactive element. Only a small portion of the page may be visible to an individual using a screen magnifier at a high level of magnification. Such a user may interpret a field in the wrong context if the focus order is not logical.

## Suggested resolution:
After navigating back to the previous screen, the focus should return to the element that triggered the navigation.
https://appt.org/en/docs/react-native/samples/accessibility-focus

## Environment:
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

https://github.com/user-attachments/assets/a96505b9-4b02-43e7-b52d-597ad5b81642

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7008074
**Issue reported by:** NA
**Slack conversation:** NA
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022016982924872499877</li>
        <li>Upwork Job ID: 2016982924872499877</li>
        <li>Last Price Increase: 2026-02-05</li>
    </ul>
</details>

**Comment by rushatgabhane** [comment]
Please feel free to use Melvin's analysis

**Comment by daledah** [comment]
i'll give an update on Monday

**Comment by marufsharifi** [comment]
@mkhutornyi, could you please take a look at my proposal when you get a chance? thanks.

**Comment by roryabraham** [comment]
Thanks, assigned to you @marufsharifi

**Comment by rushatgabhane** [comment]
We're working on a holistic fix for this and two other issues in 

- https://github.com/Expensify/App/issues/86699

**Comment by rushatgabhane** [comment]
@TaduJR any progress here?

**Comment by TaduJR** [comment]
We use shared components from https://github.com/Expensify/App/issues/76944

The last PR is getting reviewed

**Comment by rushatgabhane** [comment]
@TaduJR what's latest here?

**Comment by TaduJR** [comment]
@rushatgabhane 

WIP.

**Comment by rushatgabhane** [comment]
@roryabraham please remove hold from title when you can : )

**Comment by marufsharifi** [comment]
@rushatgabhane, just to confirm, should I continue with this now that the issue is no longer on hold? Thanks!

**Comment by marufsharifi** [comment]
@rushatgabhane, could you please take a look at [this](https://github.com/Expensify/App/issues/77411#issuecomment-4457653263) when you get a chance? thanks.

**Comment by rushatgabhane** [comment]
Hey, we're moving with a different solution as mentioned here https://github.com/Expensify/App/issues/77411#issuecomment-4158970949

**Comment by TaduJR** [comment]
PR is ready.

**Comment by trjExpensify** [comment]
approved $1,750 for @mkhutornyi
